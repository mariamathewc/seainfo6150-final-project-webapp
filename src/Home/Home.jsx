import React from 'react'
import Form from "../Form/Form";
import treefrog from "../images/treefrog.jpg";

const Home = () => {
    return (
        <div>
            The home page
            <h1>I changed this homepage</h1>
            <img src={treefrog} alt="tree frog" />
            <Form />
        </div>
    )
}

export default Home
