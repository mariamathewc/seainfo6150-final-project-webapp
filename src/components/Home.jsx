import React from 'react';
import styles from "./Home.module.css";
import { NavLink } from 'react-router-dom';


	const Home = (props) => {
	

        return (




			<div className={styles.container}>
				<div className={styles.home}>
					<h2>Travel in Tents</h2>
					<br />
					<p>Welcome to Travel in Tents</p>
					<p>Jump right in and explore our campgrounds</p>
					<p> Checkout our exclusive outdoor equipments to make your camping fun</p>

					<NavLink className={styles.footer} to='/about' exact activeClassName='active'  > <b>About </b> </NavLink>
					<NavLink className={styles.footer} to='/contacts' exact activeClassName='active'  > <b>Contacts </b> </NavLink>


				</div>
			</div>

            

		)
	
}

export default Home 