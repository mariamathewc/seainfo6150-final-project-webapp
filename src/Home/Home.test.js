import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home.jsx";

it("renders the home component correctly", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
