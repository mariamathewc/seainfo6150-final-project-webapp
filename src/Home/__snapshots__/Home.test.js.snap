// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the home component correctly 1`] = `
<div>
  <div>
    The home page
    <img
      alt="tree frog"
      src="/images/treefrog.jpg"
    />
    <div>
      <form>
        <label
          for="myTextId"
        >
          Input
        </label>
        <input
          id="myTextId"
          name="myText"
          type="text"
        />
        <label
          for="myDropdownId"
        >
          Input
        </label>
        <select
          id="myDropdownId"
          name="myDropdown"
        >
          <option
            value="foo"
          >
            Foo
          </option>
          <option
            value="bar"
          >
            Bar
          </option>
          <option
            value="baz"
          >
            Baz
          </option>
        </select>
        <input
          type="submit"
          value="Send it"
        />
      </form>
    </div>
  </div>
</div>
`;
